corehq/apps/hqmedia/tests/test_manage_paths.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+import json
+from django.test import SimpleTestCase
+import os
+
+from mock import patch
+
+from corehq.apps.app_manager.tests.app_factory import AppFactory
+from corehq.apps.app_manager.tests.util import TestXmlMixin
+from corehq.apps.hqmedia.view_helpers import download_multimedia_paths_rows
+
+
+class ManagePathsTest(SimpleTestCase, TestXmlMixin):
+    file_path = ('data',)
+
+    def test_paths_download(self):
+        factory = AppFactory(build_version='2.40.0')
+        app = factory.app
+        app.langs = ['en', 'fra']
+
+        module, form = factory.new_basic_module('register', 'case')
+        form.source = self.get_xml('one_question_two_images').decode('utf-8')
```

Comment:
@nickpell This `decode` smells bad, but without it, this assignment fails because of [this check](https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/app_manager/models.py#L734-L737), since the data I'm passing in is [already unicode](https://github.com/dimagi/commcare-hq/blob/master/corehq/util/test_utils.py#L113). Any idea if I'm doing something wrong here?

Reply:
got it, thanks